data/teams.js

```diff
@@ -2216,7 +2216,7 @@ var teams = [
    {"team_number": 5288, "lat": 43.044, "lng":  -81.239},
    {"team_number": 5290, "lat": 45.269, "lng":  -92.975},
    {"team_number": 5291, "lat": 29.558, "lng":   34.948},
-   {"team_number": 5293, "lat": 33.984, "lng":  -84.153},
+   {"team_number": 5293, "lat": 33.010, "lng":  -84.144},
```

Comment:
This PR still has the old changes. I recommend doing this:
```sh
git reset HEAD^  # undo the previous commit, but keep changes
git stash save  # stash the changes away for later
git reset --hard HEAD^  # delete the first commit
git stash pop  # reapply the changes you stashed
# add, commit, and push
```
I think there's a more efficient way to do this, but that's the first thing that came to mind.

Reply:
You could also use `git cherry-pick` I think.